akka/README.md

```diff
@@ -0,0 +1,3 @@
+# Lightbend Telemetry: Akka samples
+
+The samples here a based on akka-sample-cqrs samples in <https://github.com/akka/akka-samples/>.
```

Comment:
It would be better to base this effort on the tutorial sample in the platform guide. It will replace the cqrs sample.

https://github.com/akka/akka-platform-guide/tree/main/docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-scala

https://github.com/akka/akka-platform-guide/tree/main/docs-source/docs/modules/microservices-tutorial/examples/shopping-cart-service-java

Reply:
Great, I will do that. Thanks for the heads up, @patriknw. :-)